Implement property for Command.action. (#2433)
beeware/toga: changes/2433.feature.rst

> It's now easier to change the action for a Toga command to point to a different handler without having to jump through hoops to create the correct ActionHandler.

Reviewer: "jump through hoops" is a little pejorative (and an idiom that might not be clear to some readers):
```suggestion
An action for a Toga command can now be easily modified after initial construction.
```
